gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -102,6 +102,8 @@ enum
   PROP_EPOCHS,                  /* Repetitions of training */
 };
 
+#define TRAINER_TENSOR_SIZE_LIMT NNS_TENSOR_SIZE_LIMIT + NNS_TENSOR_SIZE_EXTRA_LIMIT
```

Comment:
To avoid unexpected results in the macro, would it be better to add parentheses as shown below?"

```suggestion
#define TRAINER_TENSOR_SIZE_LIMT ((NNS_TENSOR_SIZE_LIMIT) + (NNS_TENSOR_SIZE_EXTRA_LIMIT))
```

Reply:
Okay, thank you.